### PR TITLE
ADEN-5530 Ooyala pre-rolls tests for news&stories

### PR DESCRIPTION
**Pull request by nandy-andy**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomOoyala.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    WebElement playButton = driver.findElement(By.cssSelector(PLAY_BUTTON_SELECTOR));
+
+    wait.forElementVisible(playButton);
+    playButton.click();
+
+    verifyColorAd(driver.findElement(By.cssSelector(PLAYER_SELECTOR)), BLUE, AD_DURATION_SEC);
+  }
+
+  public void testOoyalaAutoplayPreroll() {
+    loadPage("orphan-black-clones-names");
```
Comment (fraszczakszymon) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomOoyala.java]: Can we move it to data provider?

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomOoyala.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  )
+  @Test(
+      groups = {"AdsFandomOoyalaAutoplayPrerollMobile"}
+  )
+  public void adsFandomOoyalaAutoplayPrerollMobile() {
+    testOoyalaAutoplayPreroll();
+  }
+
+  public void testOoyalaClickToPlayPreroll() {
+    loadPage("the-best-movies-of-2017-so-far");
```
Comment (fraszczakszymon) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomOoyala.java]: Can we move it to data provider?

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomOoyala.java (hunk 1 above)
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomOoyala.java]: Discussed offline: let's keep it like that and if it duplicates we'll use `const`.

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsOoyalaObject.java [hunk 3]
```diff
@@ -1,9 +1,11 @@
 package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase;
 
-import com.wikia.webdriver.common.core.CommonExpectedConditions;
 import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.OoyalaPrerollAd;
+
 import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.helpers.AdsComparison;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsOoyalaObject.java]: MINOR Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.helpers.AdsComparison'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsOoyalaObject.java#L16](https://github.com/Wikia/selenium-tests/blob/21cac82d2725fc355533db3979d4005e99fea3cd/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsOoyalaObject.java#L16): Remove this unused import 'java.util.concurrent.TimeUnit'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsOoyalaObject.java#L16](https://github.com/Wikia/selenium-tests/blob/fbaeb9e257c75fef9dcf0cec2b6ff68621ae72d1/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsOoyalaObject.java#L16): Remove this unused import 'java.util.concurrent.TimeUnit'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**